public_html/wp-content/plugins/wc-post-types/inc/favorite-schedule-shortcode.php

```diff
@@ -164,9 +164,11 @@ function preprocess_schedule_attributes( $attr ) {
 		array(
 			'date'         => null,
 			'tracks'       => 'all',
-			'speaker_link' => 'anchor',    // anchor|wporg|permalink|none
-			'session_link' => 'permalink', // permalink|anchor|none
-		), $attr
+			// Default to anchor links only if we're not using blocks on this site.
```

Comment:
`SHOULD` - I don't think this comment needs to explain _what_ the statement is doing, because that's fairly easy to see. Instead, I think it should explain _why_ it's doing it.

Although, explaining what the `content_blocks` feature flag does would be good, since that's not obvious from the name.